src/libertem/executor/base.py

```diff
@@ -37,6 +55,20 @@ def ensure_async(self, pool=None):
         """
         return AsyncAdapter(wrapped=self, pool=pool)
 
+    def run_process_local(self, task: GenericTaskProtocol, args=(), kwargs={}):
+        """
+        run a callable :code:`fn` in the context of the current process.
+        """
+        env = self._get_local_env()
+        return task(args, kwargs, environment=env)
```

Comment:
Small issue here: the `kwargs` are a quasi-global variable; if a first task is called with the default `kwargs` and calls, for example, `kwargs.update(...)`, the next calls of `run_process_local` with default `kwargs` will get the updated `kwargs`, too.